packages/eslint-plugin-query/src/rules/prefer-query-object-syntax/prefer-query-object-syntax.ts

```diff
@@ -0,0 +1,118 @@
+import type { TSESLint } from '@typescript-eslint/utils'
+import { createRule } from '../../utils/create-rule'
+import { ASTUtils } from '../../utils/ast-utils'
+
+const USE_QUERY = 'useQuery'
```

Comment:
if we want this plugin to work for `solid-query` as well, we'd need to check for `createQuery` as well, right @ardeora ?

Reply:
done